### ISSUE TITLE
Option to disable haptics

### ISSUE THREAD

**Issue by edwardbeecroft** [yonat/MultiSlider#16]
Hey @yonat,

Awesome work here. It would be great to have the option to disable the haptic feedback. In our case, we want to set a step interval, but to something small (i.e. 1.0 in a slider with a range of 39.0 - 150.0). In this case, the haptic engine fires repeatedly, so we'd like to disable it.

All the best,

Ed

**Comment by yonat** [comment]
Since I don't have time for that right now, I suggest:

```swift
slider.snapStepSize = 0
slider.valueLabelFormatter.maximumFractionDigits = 0
slider.valueLabelFormatter.positiveSuffix = ".0"
```

**Comment by edwardbeecroft** [comment]
Thanks for your time - will give this a shot. I'll close this issue for now 👍

**Comment by yonat** [comment]
Finally implemented this in 1.9.3:

```swift
isHapticSnap = false
```
